nemo/collections/tts/modules/fastpitch.py

```diff
@@ -42,10 +42,15 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+from typing import List, Optional
+
 import torch
+from omegaconf import DictConfig
 
+from nemo.collections.asr.parts.utils import adapter_utils
+from nemo.collections.tts.modules.transformer import _LayerNorm
```

Comment:
Change name _LayerNorm to ConditionalLayerNorm and move the function to submodules